docs/source/getting_started/dataset_creation/dataset_creation_local_server.rst

```diff
@@ -0,0 +1,57 @@
+.. _dataset-creation-local-server:
+
+Create a dataset with a local file server
+-----------------------------------------
+
+You can configure a dataset in the Lightly Platform that streams the images
+from your local file system. This makes the creation of a dataset much faster,
+as no upload of images and thumbnails is needd.
+Let's assume that your dataset is locally at `/projects/animals`.
+
+
+Setting up local file server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to the directory your dataset lies.
+From within this directory, create a local http server for the files:
+
+.. code-block:: bash
+
+    cd /projects/animals
+    http://localhost:1234/
+
+If the port 1234 is used, try another. You must keep the file server
+open while using the Lightly Platform.
+
+Configuring Lightly Dataset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create and configure a dataset
+
+1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly
+2. Edit your dataset and select the datasource `Local`.
```

Comment:
I find it not evident how to get to the Edit wizard. Maybe show an image where you have to click to open it?

Reply:
We don't have it in the other tutorials either and we plan to rework that soon anyway. Thus I would currently leave it like that.

Reply:
I added a red rectangle to the screenshot.